ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -626,28 +643,32 @@ public class BottomSheetController: UIViewController {
                 strongSelf.handleCompletedStateChange(to: targetExpansionState, interaction: interaction, shouldNotifyDelegate: shouldNotifyDelegate)
             } else if finalPosition == .start {
                 strongSelf.bottomSheetView.isHidden = originalBottomSheetHiddenState
-                strongSelf.bottomSheetOffsetConstraint.constant = originalBottomOffsetConstant
-            } else {
-                // The constraint constant doesn't animate, so we need to set it to whatever it should be
-                // based on the frame calculated during the interrupted animation
-                let offsetFromBottom = strongSelf.view.frame.height - strongSelf.bottomSheetView.frame.origin.y - strongSelf.view.safeAreaInsets.bottom
-                strongSelf.bottomSheetOffsetConstraint.constant = -offsetFromBottom
+                strongSelf.currentExpansionState = originalExpansionState
             }
         })
+
+        currentExpansionState = .intermediate
         translationAnimator.pauseAnimation()
         return translationAnimator
     }
 
+    // Vertical offset of bottomSheetView.origin for the given expansion state
+    //
+    // Note: Since .intermediate state doesn't have a well defined offset, this function will
+    // only return the current sheet offset in that case.
     private func offset(for expansionState: BottomSheetExpansionState) -> CGFloat {
         var offset: CGFloat
```

Comment:
Change offset to let?

Reply:
I can't believe I didn't know (or forgot) that local let variables don't need to be initialized in line :D